### DISCUSSION TITLE
How do I set the game name in Dystopia?

### DISCUSSION THREAD

**Discussion by fstltna** [GameServerManagers/LinuxGSM#4194] [Q&A]
I have looked everywhere but don't see how to update the game name from LinuxGSM to my desired game name. Where do I set it?

**Comment by dgibbs64** [comment]
As per the website https://linuxgsm.com/servers/dysserver/
`./linuxgsm.sh dysserver`
Hope that helps

**Reply by fstltna** [reply]
thanks, but that just makes another server info and doesn't change the server name of either server.
